mpl_sphinx_theme/__init__.py

```diff
@@ -3,6 +3,30 @@
 from pathlib import Path
 
 
+def set_config_defaults(app):
+
+    try:
+        theme = app.builder.theme_options
+    except AttributeError:
+        theme = None
+    if not theme:
+        theme = {}
+
+    # Default logo
+    logo = theme.get("logo", {})
+    if "image_dark" not in logo:
+        logo["image_dark"] = "_static/logo_dark.svg"
+    if "image_light" not in logo:
+        logo["image_light"] = "_static/logo2.svg"
+    if "link" not in logo:
+        logo["link"] = "https://matplotlib.org/stable/"
+    theme["logo"] = logo
+
+    # Clean up actions
+    # Update the HTML theme config
+    app.builder.theme_options = theme
```

Comment:
There are no actions?
```suggestion
    # Update the HTML theme config
    app.builder.theme_options = theme
```

Reply:
Fixed